src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -366,6 +366,16 @@ export default {
       delete extra_obj.assetsSummary.schemaKey;
       delete extra_obj.assetsSummary.numberOfBytes;
       delete extra_obj.assetsSummary.numberOfFiles;
+      _.forEach(extra_obj, (val) => {
+        if (Array.isArray(val) && val.length !== 0) {
+          val.forEach((each_val) => {
+            if (Object.prototype.hasOwnProperty.call(each_val, 'schemaKey')) {
+              // eslint-disable-next-line no-param-reassign
+              delete each_val.schemaKey;
```

Comment:
This rule doesn't seem to apply here. Do you get an error when you remove it?

Reply:
yes. and i found this [documentation](https://eslint.org/docs/rules/no-param-reassign) . so perhaps it's ok to disable the rule?

Reply:
It looks like this [more specific version](https://eslint.org/docs/rules/no-param-reassign#props) is the thing you want here.

If you can verify that and you change it here, you can merge without another review.

Reply:
did you mean that I should write it this way:
```
/* eslint no-param-reassign:["error", {"ignorePropertyModificationsFor":["each_val"] }] */
delete each_val.schemaKey;
```
?